csrc/custom_all_reduce_test.cu

```diff
@@ -330,10 +330,17 @@ int main(int argc, char** argv) {
   //     run<half>(myRank, nRanks, comm, threads, block_limit, 4096 * 1024);
   //   }
   // }
+#ifdef USE _ROCM
```

Comment:
I understand this is fixed in #137, but this  should easily not pass review. Let's spend more time ensuring code quality and that tests pass before merging.